Index: src/homework7/Main.java
===================================================================
diff --git a/src/homework7/Main.java b/src/homework7/Main.java
deleted file mode 100644
--- a/src/homework7/Main.java	(revision 442cb90d007f4fb48bd2032bd6071aba19f15dac)
+++ /dev/null	(revision 442cb90d007f4fb48bd2032bd6071aba19f15dac)
@@ -1,42 +0,0 @@
-package homework7;
-
-import java.lang.reflect.Array;
-import java.util.HashMap;
-import java.util.Map;
-
-public class Main {
-    public static void main(String[] args) {
-        String[] array = createGenericArray(String.class, 10);
-        array[0] = "cati";
-        array[1] = "dog";
-        array[2] = "cat";
-        array[3] = "lion";
-        array[4] = "dog";
-        array[5] = "dogi";
-        array[6] = "dogi";
-        array[7] = "dog";
-        array[8] = "dog";
-        array[9] = "lion";
-        arrayToMap(array);
-    }
-
-    public static <K> K[] createGenericArray(Class<K> clazz, int capacity) {
-        return (K[]) Array.newInstance(clazz, capacity);
-    }
-
-    public static <K> Map<K, Integer> arrayToMap(K[] ks) {
-        Map<K, Integer> map = new HashMap<>();
-        int numberOfOccurrences = 0;
-        for (int i = 0; i < ks.length; i++) {
-            for (int j = 0; j < ks.length; j++) {
-                if (ks[i].equals(ks[j])) {
-                    numberOfOccurrences = numberOfOccurrences + 1;
-                }
-            }
-            map.put(ks[i], numberOfOccurrences);
-            numberOfOccurrences = 0;
-        }
-        System.out.println(map);
-        return map;
-    }
-}
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1665092004025)
+++ b/.idea/vcs.xml	(date 1665092004025)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$/.." vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1665091870325)
+++ b/.idea/misc.xml	(date 1665091870325)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
\ No newline at end of file
